components-core/src/main/java/com/airwallex/android/core/ClientSecretRepository.kt

```diff
@@ -23,6 +23,17 @@ class ClientSecretRepository(private val provider: ClientSecretProvider) {
         }
     }
 
+    @Suppress("SwallowedException", "TooGenericExceptionThrown", "TooGenericExceptionCaught")
+    suspend fun retrieveClientSecret(customerId: String): ClientSecret {
+        return withContext(Dispatchers.IO) {
+            try {
+                provider.provideClientSecret(customerId)
+            } catch (e: Exception) {
```

Comment:
Again, might need to use Throwable

Reply:
And should we throw a more concrete exception? Or should we just not catch it, just let it go through?